src/client/components/pages/Schedule/ScheduleView.tsx

```diff
@@ -190,7 +202,9 @@ const ScheduleView: FunctionComponent<ScheduleViewProps> = ({
                     return (
                       <CourseListing key={meetingId}>
                         <CourseListingButton
-                          isHighlighted={popoverInBlock && isSelected}
+                          isHighlighted={(popoverInBlock && isSelected)
```

Comment:
I missed this on the first go around, but it looks like there are still some highlighting/fading styles being applied relative to the parent course, e.g. when AC209B is clicked, the other courses in CS109B's block are faded out, but not any of the other courses in other blocks.

![2023-03-20_143633](https://user-images.githubusercontent.com/6124793/226435267-ed229fb7-2722-45e7-ba1c-e81c65bd1a7f.png)

Reply:
Thanks for pointing this out! I think that this issue has been fixed via commit [5575662](https://github.com/seas-computing/course-planner/pull/652/commits/5575662160d8d10f4625549b16b0d32ca8cc7610) by adding more conditions determining the values of `popoverInBlock` and `isSelected`.

Reply:
Great! Thanks for adding the extra context around the change in the commit message (and also for cleaning up that old comment!).